Adding SCADA EtherNet/IP and CIP protocol support

kwong-solana opened the pull request:
1.  The enip-feature-1495-v13 branch contains support for the ENIP/CIP
Industrial protocol

2. This is an app layer implementation which uses the "enip" protocol
and "cip_service" and "enip_command" keywords

3.  Rebased master to clean out commits done on master and then
merged to latest code

4.  Addresses failed PR's
https://github.com/inliniac/suricata/pull/1810 and
https://github.com/inliniac/suricata/pull/1819 and
https://github.com/inliniac/suricata/pull/1827 and
https://github.com/inliniac/suricata/pull/2009 and
https://github.com/inliniac/suricata/pull/2011 and
https://github.com/inliniac/suricata/pull/2012 and
https://github.com/inliniac/suricata/pull/2019 and
https://github.com/inliniac/suricata/pull/2030 and
https://github.com/inliniac/suricata/pull/2068

5.  As requested, added AFL entry points

inliniac commented:
I rebased this but then I get a failing unittests. Could you rebase it to master and resubmit?

kwong-solana commented:
done, added new branch enip-feature-1495-v14


On 28/09/2016 4:29 AM, Victor Julien wrote:
>
> I rebased this but then I get a failing unittests. Could you rebase it 
> to master and resubmit?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/inliniac/suricata/pull/2107#issuecomment-250104366>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AMg0-rZzcIxsOTGdQ97hCQy3fkwc0wOAks5quiVMgaJpZM4ImokH>.
>

-- 
Kevin Wong

Solana Networks Inc | 613.596.2557 x575 | kwong@solananetworks.com
  
http://www.solananetworks.com

http://www.sparrowiq.com
